tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php

```diff
@@ -115,6 +116,8 @@ public function assertRootCategoriesTree(TableNode $tableNode, string $langIso):
      */
     public function assertCategoriesTree(TableNode $tableNode, string $langIso, string $parentReference): void
     {
+        Cache::clean('Category::getNestedCategories_*');
```

Comment:
This should be moved into `Category::resetStaticCache` (needs to be created) as it can be useful in other test use cases